File: Pattern3.c
//*
//**
//***
//****
//*****
//****
//***
//**
//*

#include <stdio.h>

void main()
{
    int n;
    printf("enter the row number:");
    scanf("%d",&n);

    int j = 1, k = 1, i = 1;
    for (i = 1; i <= (2 * n - 1); i++)
    {
        k = (i <= n) ? i : (2 * n - i);
        for (j = 1; j <= k; j++)
        {
            printf("*");
        }
        printf("\n");
    }
}

File: Pattern3.go
//*
//**
//***
//****
//*****
//****
//***
//**
//*


package main

import "fmt"

func main(){
	
var  n int;
fmt.Printf("Enter the row number:")
fmt.Scanln(&n)
var k int;
for i:= 1; i <= (2 * n - 1); i++ {
	if i<=n {
		k = i;
	}else{
		k = (2*n-i)
	}
    for j:= 1; j <= k; j++ {
        fmt.Printf("*");
    }
    fmt.Printf("\n")
}
}